app/pages/about/publications-page.cjsx

```diff
@@ -418,6 +418,10 @@ publicationCategories =
 module.exports = React.createClass
   displayName: 'PublicationsPage'
 
+  statics:
```

Comment:
I think instead of adding a static method here, we should extract the publication data into its own js file to live in the `lib` folder since this data is now being used in two different components.